src/workloads/query/QuerySettings.yml

```diff
@@ -0,0 +1,164 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  Measure the performance for find command with-/out query settings as well as checking both hit and miss cases.
+
+KeyWords:
+- querySettings
+- find
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 400
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection querySettingsColl
+  DocumentCount: &DocumentCount 100000
```

Comment:
Is there a value of having so many documents in the test collection? Insert of 1000s of docs takes time.

Reply:
Would you propose inserting 1000 documents instead?

Reply:
The absolute minimum, whatever it is.